### PR TITLE
SP-4405: updating hello-world -slack playbook

### PR DESCRIPTION
**Pull request by eldadru**
### General
- [x] Go over texts with @stackpulse/product

### New playbooks
- [x] Tested Playbook in a production tenant
- [x] Added a link in the [README.MD](https://github.com/stackpulse/playbooks/blob/master/README.md)

### CODE REVIEW

**Review by rzsp — APPROVED** [review]

File: hello-world/hello-world-slack/playbook.yaml [hunk 1]
```diff
@@ -18,13 +18,16 @@ steps:
...
+      MESSAGE_TEXT: "*Congratulations!* :tada:\nYou’ve successfully integrated StackPulse with Slack and triggered your first playbook.\n\n*Want to know what just happened?*\n check out the <https://app.stackpulse.io/playbook/{{ .metadata.playbookId }}|Trigger & Playbook>\n\n*What's next?*\n\n- Run playbooks automatically.  Get more <{{ .params.stackpulsePlaybooksURL }}|playbooks> from StackPulse repository\n- Enrich alerts from your monitoring and infrastructure services.  Set up more <https://app.stackpulse.io/integrations|integrations>\n- Use `/stackpulse help` to see what you can do in Slack\n- For more information, checkout our <https://docs.stackpulse.io|documentation>\n\n*What’s possible?*"
```
Comment (alongat) [inline hello-world/hello-world-slack/playbook.yaml:26]:
```suggestion
      MESSAGE_TEXT: "*Congratulations!* :tada:\nYou’ve successfully integrated StackPulse with Slack and triggered your first playbook.\n\n*Want to know what just happened?*\n check out the <https://app.stackpulse.io/playbook/{{ $.metadata.playbookId }}|Trigger & Playbook>\n\n*What's next?*\n\n- Run playbooks automatically.  Get more <{{ .params.stackpulsePlaybooksURL }}|playbooks> from StackPulse repository\n- Enrich alerts from your monitoring and infrastructure services.  Set up more <https://app.stackpulse.io/integrations|integrations>\n- Use `/stackpulse help` to see what you can do in Slack\n- For more information, checkout our <https://docs.stackpulse.io|documentation>\n\n*What’s possible?*"
```

Comment (alongat) [reply hello-world/hello-world-slack/playbook.yaml:26]: there's another one above

**Review by alongat — APPROVED** [review]
See comments about . -> $.

File: hello-world/hello-world-slack/playbook.yaml (hunk 1 above)
Comment (eldadru) [reply hello-world/hello-world-slack/playbook.yaml:26]: Great, adding

**Pull request merged**